test/e2e/fixture/app/consequences.go

```diff
@@ -78,12 +76,3 @@ func (c *Consequences) resource(kind, name string) ResourceStatus {
 		},
 	}
 }
-
```

Comment:
Is this change intentional? We don't need `ARGOCD_E2E_EXPECT_TIMEOUT` any more?

Reply:
not needed